website/src/pages/news.tsx

```diff
@@ -0,0 +1,103 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import { ContentSection } from '../components/content/ContentSection'
+import Layout from '../components/Layout'
+export default class NewsPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            events: [],
+        }
+    }
+
+    public componentDidMount(): void {
+        if (document) {
+            document.getElementsByTagName('body')[0].setAttribute('style', 'background-image:none')
+        }
+    }
+
+    public render(): JSX.Element | null {
+        const desc =
+            'Sourcegraph in the news.'
+        return (
+            <Layout location={this.props.location}>
+                <div className="News bg-white text-dark">
+                    <Helmet>
+                        <title>Sourcegraph - News</title>
+                        <meta name="twitter:title" content="Sourcegraph news" />
+                        <meta property="og:title" content="Sourcegraph news" />
+                        <meta name="description" content="Sourcegraph news" />
+                        <meta name="twitter:description" content="Sourcegraph news" />
+                        <meta property="og:description" content="Sourcegraph news" />
+                    </Helmet>
+                    <section className="news news__head">
+                        <div className="news__container">
+                        <ContentSection color="primary" className="hero-section text-center py-5">
+                            <h1>Sourcegraph in the news</h1>
+                            <p className="news__head-description">
+                                The latest news and press releases.
+                            </p>
+                        </ContentSection>
+                        </div>
+                    </section>
+                    <section className="newsroom">
+                        <div class="container">
+                            <div class="row justify-content-md-center">
+                                <div class="col mt-5">
+                                    <h3>Press Releases</h3>
```

Comment:
```suggestion
                                    <h3>Press releases</h3>
```

per our style guide (https://about.sourcegraph.com/handbook/communication/style_guide#general): "Use 'Sentence case'"